source/code-generation/query-converter/test-converted-queries/compare-converted-queries.txt

```diff
@@ -1,13 +1,190 @@
-Compare Query Results
-=====================
+=========================
+Compare Converted Queries
+=========================
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Task page.
\ No newline at end of file
+You can use the :guilabel:`Test Queries` pane to compare source 
+and destination queries, views, and stored procedure results 
+inside of Relational Migrator. The :guilabel:`Test Queries` can 
+help you verify the accuracy of converted code and
+show the source and destination data after you run a sync job.
+
+About this Task
+---------------
+
+The :guilabel:`Test Queries` pane is split into three separate 
+user interfaces. Click the pill buttons next to 
+:guilabel:`Test Queries` to change the user interface. The 
+following table summarizes each user interface:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 80
+
+   * - User Interface 
+     - Description
+   * - Console
+     - This is the default user interface of the 
+       :guilabel:`Test Queries` pane. You can click 
+       the :guilabel:`Run Source Query` button to 
+       run all converted queries. You can also specify 
+       parameters for stored procedures in the 
+       :guilabel:`Source Script` text field.
+   * - Results
+     - This user interface is used to view the query results 
+       from the source and destination queries. You must run 
+       a source or destination query to view the ``Results``
+       user interface.
+   * - Messages
+     - This user interface is used to show messages from each 
+       database connection. You can see error messages, 
+       execution stats, and any print statements.
+
+Before you Begin
+----------------
+
+- You must sign in to your Atlas account in Relational Migrator. 
+  For details, see :ref:`rm-atlas-log-in`.
+
+- For details on how to turn on Query Converter, see 
+  :ref:`rm-enable-query-converter`.
+
+- To use the :guilabel:`Test Queries` pane, Relational
+  Migrator must have access to the Query Runner. For details
+  on how to setup Query Runner, see :ref:`rm-install-query-runner`.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Select a query on the :guilabel:`Query Converter` pane
+
+      a. From the :guilabel:`Code Generation` tab, click the 
+         :guilabel:`Query Converter` pane.
+
+         .. note::
+
+           If you are not already logged into your Atlas account or have
+           an expired session, you must login to proceed.
+
+      #. On the left-hand :guilabel:`Queries` pane, select a query, view,
+         or stored procedure.
+
+      #. If your query has not been converted, select a 
+         :guilabel:`Target Language` and click :guilabel:`Convert`
+         :icon-lg:`Sparkle`.
+
+   .. step:: Open the :guilabel:`Test Queries` pane
+
+      a. From the :guilabel:`Code Generation` tab, click the 
+         :guilabel:`Query Converter` pane.
+
+      #. Click the :icon-lg:`ChevronUp` button on the 
```

Comment:
[Suggestion]

Maybe to help users orient themselves a little quicker "On the bottom-right of the screen, Click the :icon-lg:`ChevronUp` button on the .."